Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -3703,6 +3704,12 @@ private unsafe void OnUpdate(InputUpdateType updateType, ref InputEventBuffer ev
             ////       mess in the event buffer
             ////       same goes for events that someone may queue from a change monitor callback
             InvokeAfterUpdateCallback(updateType);
+            //send pointer data to backend for OnMouseEvents
+#if UNITY_INPUTSYSTEM_HAS_MOUSE_SCRIPT_EVENTS_SUPPORTED
```

Comment:
As suggested by @K-Tone 
```suggestion
#if UNITY_INPUTSYSTEM_SUPPORTS_MOUSE_SCRIPT_EVENTS
```